### PR TITLE
[AWS] [ALB] Add ALB traffic on ELB log dashboard

### LINKED ISSUE #10149: [AWS] AWS ELB metrics integration OOTB Dashboard does not reflect ALBs data for Inbound and Outbound Traffic Visualisations
The Visualisations for Inbound and Outbound traffic in this OOTB dashboard utilises data from `source.bytes` and `destination.bytes` fields. 

The integration's [ingest pipeline](https://github.com/elastic/integrations/blob/dfc057e4908d28fbe9f676ee4e3e0bdbe67cd35b/packages/aws/data_stream/elb_logs/elasticsearch/ingest_pipeline/default.yml) is currently only parsing and populating these fields for TCP Logs from Classic ELB & NLB.

It seems the values parsed from ALB logs to populate `http.request.body.bytes` and `http.response.body.bytes` fields should also be used to populate `source.bytes` and `destination.bytes` fields to address this issue.

### PR DESCRIPTION
**Pull request by anil-elastic**
## Proposed commit message
Add Inbound and Outbound Traffic data for ALB on ELB log dashboard. ALB logs to populate http.request.body.bytes and http.response.body.bytes fields.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Closes #10149 
## Screenshots
Screenshot_2024-07-12_12-19-23

**Removed square brackets around ALB ( review comment)**
filebeat-aws-elb-overview

### CODE REVIEW

**Comment by agithomas** [comment]
Please remove the square brackets around `ALB`

**Comment by anil-elastic** [comment]
> Please remove the square brackets around `ALB`
@agithomas Updated the panel. Please review.

**Comment by agithomas** [comment]
You may have to update the dashboard screenshot file - `img/filebeat-aws-elb-overview.png`

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.19.0"
+  changes:
+    - description: Add Inbound and Outbound Traffic data for ALB on ELB log dashboard.
```
Comment (agithomas) [inline packages/aws/changelog.yml:4]: Data do exist, I think, what you are adding here is the visualisation panel.

File: packages/aws/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+      type: enhancement
+      link: https://github.com/elastic/integrations/issues/10119
```
Comment (agithomas) [inline packages/aws/changelog.yml:6]: Suggest to replace with PR link.

File: packages/aws/changelog.yml (hunk 1 above)
Comment (agithomas) [inline packages/aws/changelog.yml:4]:
```suggestion
    - description: Add a visualization panel to display the Inbound and Outbound traffic of Application load balancers. 
```

**Review by mbondyra — APPROVED** [review]
LGTM 🆗

**Review by gpop63 — APPROVED** [review]

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**